### PR TITLE
Stunner - Add Intermediate Timer Event properties

### PR DESCRIPTION
**Pull request by jlindop**

### CODE REVIEW

**Comment by jlindop** [comment]
jenkins retest this

**Comment by jlindop** [comment]
jenkins please retest this

**Comment by tsurdilo** [comment]
+1

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/test/java/org/kie/workbench/common/stunner/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java [hunk 1]
```diff
@@ -686,6 +687,31 @@ public void testMarshallXorGateway() throws Exception {
         assertTrue( result.contains( "<bpmn2:exclusiveGateway id=\"_877EA035-1A14-42E9-8CAA-43E9BF908C70\" drools:dg=\"under 10 : _5110D608-BDAD-47BF-A3F9-E1DBE43ED7CD\" name=\"AgeSplit\" gatewayDirection=\"Diverging\" default=\"_5110D608-BDAD-47BF-A3F9-E1DBE43ED7CD\">" ) );
     }
 
+    public void testMarshallIntermediateTimerEvent() throws Exception {
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/test/java/org/kie/workbench/common/stunner/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java]: Should this be annotated ```@Test```?

Comment (jlindop) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/test/java/org/kie/workbench/common/stunner/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java]: Thanks @manstis. This is now fixed.

**Comment by manstis** [comment]
Thanks @jlindop I'll wait for Jenkins to give this the green light again (before merging).

Please do however think about using JIRAs now for Stunner changes - IMO it was fine committing ad-hoc when you were using @romartin fork, but now this is in "core" I think we should use JIRA references in the commits.

**Comment by jlindop** [comment]
@manstis Yes, I agree that using JIRA's makes sense now that we're working in kie-wb-common.

**Pull request merged**